ds3-sdk/src/test/java/com/spectralogic/ds3client/metadata/MetaDataAccessImplModelTest.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
When I ran the unit tests with coverage, only 3 classes had any coverage.  Of those that had coverage, the most method coverage was 20%